Engines/Wine/Verbs/dotnet40/script.js

```diff
@@ -1,14 +1,20 @@
 include(["engines", "wine", "engine", "object"]);
 include(["engines", "wine", "plugins", "override_dll"]);
 include(["utils", "functions", "net", "resource"]);
-include(["engines", "wine", "verbs", "luna"]);
-include(["utils", "functions", "filesystem", "files"]);
+include(["engines", "wine", "plugins", "regedit"]);
+include(["engines", "wine", "plugins", "windows_version"]);
 
 /**
 * Verb to install .NET 4.0
 * @returns {Wine} Wine object
 */
 Wine.prototype.dotnet40 = function () {
+    if (this.architecture == "amd64") {
```

Comment:
Are you sure about `this.architecture` and not `this.architecture()`? `Wine.prototype.architecture` is a function, see:
https://github.com/PhoenicisOrg/scripts/blob/f7c70527bc69dab5f0aa8bef49fda2674d9e6575/Engines/Wine/Engine/Object/script.js#L29-L43

Reply:
This needs to be changed in the other files too.